docs/tutorials/go.md

```diff
@@ -367,7 +367,11 @@ func main() {
 
  flag.Parse()
 
- logger := log.MustNewDefaultLogger(log.LogFormatPlain, log.LogLevelInfo, false)
+ logger, err := log.NewDefaultLogger(log.LogFormatPlain, log.LogLevelInfo, false)
+ if err != nil {
```

Comment:
Did gofmt approve this?

Reply:
it's in a markdown file

Reply:
Ha. I often use play.golang.org to format samples for MD, FWIW.